### PR TITLE
openpgp.js: Add HKP types

### PR DESCRIPTION
**Pull request by errietta**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/openpgpjs/openpgpjs/blob/master/src/hkp.js
- [x] Increase the version number in the header if appropriate.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

File: types/openpgp/openpgp-tests.ts [hunk 1]
```diff
@@ -135,6 +135,23 @@ openpgp.initWorker({ path:'openpgp.worker.js' });
     return verified.signatures[0].valid;
 })();
 
+async () => {
+    let hkp = new openpgp.HKP();
+
+    let hkpOptions = {
```
Comment (po5i) [inline types/openpgp/openpgp-tests.ts]: I think we should use `const` instead. What do you think?

**Review by po5i — CHANGES_REQUESTED** [review]

File: types/openpgp/openpgp-tests.ts (hunk 1 above)
Comment (errietta) [reply types/openpgp/openpgp-tests.ts]: good point, changed

**Comment by errietta** [comment]
@po5i can you approve if you're ok with this?

**Review by po5i — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**